### ISSUE TITLE
How much loss should one expect at 200,000 iterations?

### ISSUE THREAD

**Issue by sourabh-patil** [Mukosame/Zooming-Slow-Mo-CVPR-2020#36]
Hi team,
I am training the model as per given instructions. At the starting point loss in the range of 10 power 5. Even after training the model for more than 200,000 iterations, the loss has decreased 10 times only (its in the range of 10 power 4). I am uploading 2 log files, kindly look at them. While training, deformable convolutions' offset seem to go above certain limit, but after some iterations it became normal (You can observe this in 2nd log at around166,000th iteration
[1st.log](https://github.com/Mukosame/Zooming-Slow-Mo-CVPR-2020/files/5137027/1st.log)
[2nd.log](https://github.com/Mukosame/Zooming-Slow-Mo-CVPR-2020/files/5137028/2nd.log)

).

**Comment by Mukosame** [comment]
Hi @sourabh-patil , the loss should be always fluctuate a lot during the later training phase. Your loss looks fine for me.
Besides, it is a dangerous alert when the offset is too high --- which means the model doesn't converge. It's good if it goes back to normal.

**Comment by sourabh-patil** [comment]
Thank you for your reply. Please can you tell me an approximate range of values of loss when the model finally converges (around 600,000 iterations right?!).

**Comment by Mukosame** [comment]
Hi Sourabh, it denpends on your batch size. The model actually converges every 150,000 iterations since we set periodic learning rate in the option file. Your range looks correct. Comparing to the training loss, it's better to tell if the model converges by testing on a validation set in every checkpoint. The PSNR values are more stable than the loss.

**Comment by sourabh-patil** [comment]
Oh I see. Thank you for the explanation.
